src/Cli/dotnet/dotnet.csproj

```diff
@@ -95,6 +96,7 @@
     <PackageReference Include="Microsoft.Deployment.DotNet.Releases" Version="$(MicrosoftDeploymentDotNetReleasesVersion)" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="$(SystemServiceProcessServiceControllerVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="$(MicrosoftVisualStudioSetupConfigurationInteropVersion)" PrivateAssets="All" ExcludeAssets="Runtime"/>
+    <PackageReference Include="Sprache" Version="$(SpracheVersion)" />
```

Comment:
@MichaelSimons is Sprache source build compliant? This is only for 6.0.4xx so maybe not as risky if source build stays on 6.0.1xx.

The license is MIT.  @richlander @leecow on the question of process for pulling in a new open source library.

Reply:
Even if this is only going into 6.0.4xx it still would impact source-build when it reached 7.0.

Sprache is not something included in source-build today.  I would suggest reading through the [documentation](https://github.com/dotnet/source-build/blob/main/Documentation/sourcebuild-in-repos/new-dependencies.md) on adding new source-build dependencies.  It appears to only have a [dependency on source-link](https://github.com/sprache/Sprache/blob/develop/src/Sprache/Sprache.csproj) therefore it could be handled via an [external repository that we source-build](https://github.com/dotnet/source-build-externals).

Reply:
I sidestepped the source-build conversation by removing Sprache in favor of a hand-rolled parser.